caluma/caluma_form/tests/test_answer.py

```diff
@@ -252,7 +253,7 @@ def test_save_default_answer_python_api(
         answer = api.save_default_answer(question, user=admin_user, value=inp["value"])
         snapshot.assert_match(answer)
     else:
-        with pytest.raises(CustomValidationError):
+        with pytest.raises((CustomValidationError, ValidationError)):
```

Comment:
You can ditch the `CustomValidationError` here, as it's a subclass of DRF's `ValidationError`.

Reply:
done